report/finding.go

```diff
@@ -13,7 +13,7 @@ type Finding struct {
 	StartColumn int
 	EndColumn   int
 
-	Line string `json:"-"`
+	Lines string `json:"-"`
```

Comment:
Should this change?

Reply:
I intentionally changed the name to better reflect what the struct holds. When it finds a match, the field contains all of the lines the match was found on, not just the first line.